pyexchange/erisx.py

```diff
@@ -72,7 +72,7 @@ class ErisxApi(PyexAPI):
 
     def __init__(self, fix_trading_endpoint: str, fix_trading_user: str,
                  fix_marketdata_endpoint: str, fix_marketdata_user: str, password: str,
-                 clearing_url: str, api_key: str, api_secret: str):
+                 clearing_url: str, api_key: str, api_secret: str, account_id: int = 0):
```

Comment:
That makes sense; usually the consumer wants the first on the list.  This allows them to connect even if they don't know what their account_id is.